### PR TITLE
Enable Model Garden, Vision API and Vertex Standalone RAG APIs tests 

### PR DESCRIPTION
**Pull request by gmogr**

### CODE REVIEW

File: libs/vertexai/tests/integration_tests/test_gemma.py [hunk 1]
```diff
@@ -14,16 +14,16 @@
 )
 
 
-@pytest.mark.extended
+@pytest.mark.release
```
Comment (lkuligin) [inline libs/vertexai/tests/integration_tests/test_gemma.py:17]: it probably still should be extended

File: libs/community/tests/integration_tests/test_googlesearch_api.py [hunk 2]
```diff
@@ -1,32 +1,41 @@
 """Integration test for Google Search API Wrapper."""
 
-import pytest
+import os
 
 from langchain_google_community.search import GoogleSearchAPIWrapper
 
 
-@pytest.mark.skip(reason="CI/CD not ready.")
 def test_call() -> None:
```
Comment (lkuligin) [inline libs/community/tests/integration_tests/test_googlesearch_api.py:8]: should we mark it as extended (and add a corresponding flag to pytest conf)?

File: libs/vertexai/tests/integration_tests/test_model_garden.py [hunk 3]
```diff
@@ -100,7 +97,7 @@ async def test_model_garden_agenerate(
 
 @pytest.mark.xfail(reason="CI issue")
```
Comment (lkuligin) [inline libs/vertexai/tests/integration_tests/test_model_garden.py:98]: should we mark it as extended and drop xfail?

File: libs/community/tests/integration_tests/test_vision.py [hunk 4]
```diff
@@ -1,16 +1,14 @@
...
 
-import pytest
 from langchain_core.document_loaders.blob_loaders import Blob
 from langchain_core.documents import Document
 
 from langchain_google_community import CloudVisionLoader, CloudVisionParser
 
 
-@pytest.mark.skip(reason="CI/CD not ready.")
 def test_parse_image() -> None:
```
Comment (lkuligin) [inline libs/community/tests/integration_tests/test_vision.py:9]: should we mark it as extended?

File: libs/community/tests/integration_tests/test_googlesearch_api.py (hunk 2 above)
Comment (gmogr) [reply libs/community/tests/integration_tests/test_googlesearch_api.py:11]: done

**Pull request merged**